packages/grafana-schema/src/veneer/dashboard.types.ts

```diff
@@ -8,6 +8,7 @@ export type { CommonDataSourceRef as DataSourceRef };
 export interface Panel<TOptions = Record<string, unknown>, TCustomFieldConfig = Record<string, unknown>>
   extends Omit<raw.Panel, 'fieldConfig'> {
   fieldConfig: FieldConfigSource<TCustomFieldConfig>;
+  snapshotData?: unknown[];
```

Comment:
rather than keeping the same snapshot support (anythign!) can scenes require migration to a snapshot query saved in the regular place?

the migration is easy, and could simplify things greatly.

Reply:
Yup, have not thought about it at all! I'll try it out next week.

Reply:
see https://github.com/grafana/grafana/pull/54900 for reference

Reply:
Thanks @torkelo - I sort of completely forgot we have snapshot queries available ;)

Reply:
@dprokop well not sure we have, that PR was never merged. So might need some small change to grafana data source to support it:
https://github.com/grafana/grafana/pull/54900/files#diff-f1386ecf8e451a84d16e901593385dbecf86e95242c70084a41c1c43f37830a9R90

Reply:
@torkelo  - this particular change has landed in main alrady as far as I see: https://github.com/grafana/grafana/blob/dash-scenes-share-snap/public/app/plugins/datasource/grafana/datasource.ts#L104